lib/wallet.js

```diff
@@ -1886,4 +1974,220 @@ Wallet.deriveByPath = function(hdKey, path, keyPath) {
     }
 };
 
+Wallet.prototype.rskPay = function(pay, feeStrategy, twoFactorToken, options, cb) {
+    /* jshint -W071 */
+    var self = this;
+
+    if (typeof feeStrategy === "function") {
+        cb = feeStrategy;
+        feeStrategy = null;
+    } else if (typeof twoFactorToken === "function") {
+        cb = twoFactorToken;
+        twoFactorToken = null;
+    } else if (typeof options === "function") {
+        cb = options;
+        options = {};
+    }
+
+    feeStrategy = feeStrategy || Wallet.FEE_STRATEGY_OPTIMAL;
+    options = options || {};
+    var checkFee = typeof options.checkFee !== "undefined" ? options.checkFee : true;
+
+    var deferred = q.defer();
+    deferred.promise.nodeify(cb);
+
+    if (self.locked) {
+        deferred.reject(new blocktrail.WalletLockedError("Wallet needs to be unlocked to send coins"));
+        return deferred.promise;
+    }
+
+    q.nextTick(function() {
+        self.buildRskTransaction(pay, feeStrategy, options).then(
+            function(r) { return r; },
+            function(e) { deferred.reject(e); },
+        )
+            .spread(
+            function(signedTx,txid) {         
+                return self.sendRskTransaction(signedTx,txid, checkFee, twoFactorToken, options.prioboost, options)
+                    .then(function(result) {
+                        if (!result || !result['complete'] || result['complete'] === 'false') {
+                            deferred.reject(new Error("Failed to send transaction."));
+                        } else {
+                            return result['txid'];
+                        }
+                    });
+            },
+            function(e) {
+                throw e;
+            }
+        )
+            .then(
+            function(r) { deferred.resolve(r); },
+            function(e) { deferred.reject(e); }
+        )
+        ;
+    });
+
+    return deferred.promise;
+};
+
+Wallet.prototype.buildRskTransaction = function(pay,feeStrategy, options, cb) {
```

Comment:
missing documentation for pay object structure. consider breaking into arguments if it's really small?